stablehlo/tests/print_stablehlo.mlir

```diff
@@ -281,10 +281,10 @@ func.func @fft_op(%arg0: tensor<16xcomplex<f32>>) -> tensor<16xcomplex<f32>> {
 }
 
 // CHECK-LABEL: func @extensions
-func.func @extensions(%arg0 : tensor<?x?xf32, #stablehlo.type_extensions<bounds = [3, ?]>>,
+func.func @extensions(%arg0 : tensor<?x?xf32, #stablehlo.bounds<3, ?>>,
```

Comment:
We could also consider `tensor<?x?xf32, #stablehlo.bounds<3x?>>` to be consistent with how dimension sizes are prettyprinted. I'm on the fence - really don't know which one I prefer.